content/efcore/getting-started/samples/manipulating-entities/add-new-entity-to-existing/Program.cs

```diff
@@ -26,7 +26,8 @@ public static void Main()
                 .Include(b => b.Author)
                 .Single(b => b.Title.Contains("Badge"));
 
-            Console.WriteLine("{0}: {1}, {2} {3}", addedBook.Id, addedBook.Title, addedBook.Author.FirstName, addedBook.Author.LastName);
+            Console.WriteLine("Book, {0}, added to author, {1} {2} - ", addedBook.Title, author.FirstName, author.LastName);
+            Console.WriteLine("Title: {0}\n Genre: {1}\n Publication Year: {2}", addedBook.Title, addedBook.Genre, addedBook.PublicationYear);
```

Comment:
Remove the second `Console.WriteLine` that prints out the books info. It isn't necessary for this example.